### ISSUE TITLE
Issue when resizing a page

### ISSUE THREAD

**Issue by ArchangeGabriel** [pdfarranger/pdfarranger#1100]
Describe the bug
----------------

Trying to resize any page in a document results in “only pikepdf pages can be assigned to a page list; tried to assign <class 'pikepdf.objects.Object'>”.

To Reproduce
------------

1.  Open any PDF.
2.  Click on “Page format” (translated, C shortcut)
3.  Change e.g. the width.
4.  Try to save, get the error message in a pop-up window instead.

Expected behavior
-----------------

The PDF file is saved without trouble.

Screenshots
-----------

![Screenshot_20240613_065356](https://github.com/pdfarranger/pdfarranger/assets/490531/c3343460-d86d-4a4f-870c-f3b32d7cbcd3)

System and Versions
-------------------

-   PDF Arranger: 1.10.1-2 as ArchLinux distro package
-   pikepdf: 9.0.0-1 as ArchLinux distro package
-   OS: ArchLinux

**Comment by ArchangeGabriel** [comment]
This is likely similar to https://github.com/pdfarranger/pdfarranger/issues/1094, in particular
> Pdf.pages no longer coerces PDF dictionaries to page objects. You must explicitly insert/add pikepdf.Page objects

**Comment by kbengs** [comment]
Thanks, but I can't reproduce it. Can you attach a pdf which has this issue? Can you run pdfarranger from a terminal and post the error messages printed so we can see which line is failing ?

**Comment by dreua** [comment]
```
(.venv) [da@David-UB pdfarranger ((1.10.1) %)]$ pdfarranger
Traceback (most recent call last):
  File "/home/da/git/pdfarranger/.venv/lib64/python3.12/site-packages/pdfarranger/exporter.py", line 255, in wrapper
    func(*args, **kwargs)
  File "/home/da/git/pdfarranger/.venv/lib64/python3.12/site-packages/pdfarranger/exporter.py", line 387, in export
    export_doc(pdf_input, pages, mdata, files_out, quit_flag)
  File "/home/da/git/pdfarranger/.venv/lib64/python3.12/site-packages/pdfarranger/exporter.py", line 354, in export_doc
    _copy_n_transform(pdf_input, pdf_output, pages, quit_flag)
  File "/home/da/git/pdfarranger/.venv/lib64/python3.12/site-packages/pdfarranger/exporter.py", line 294, in _copy_n_transform
    pdf_output.pages[i] = _apply_geom_transform(pdf_output, pdf_output.pages[i], row)
    ~~~~~~~~~~~~~~~~^^^
TypeError: only pikepdf pages can be assigned to a page list; tried to assign <class 'pikepdf.objects.Object'>
```

I could reproduce it with 1.10.1 but not with the current main (no error). You probably changed something when implementing the new interactive resizing gui but I didn't dig deeper.

**Comment by kbengs** [comment]
The issue is fixed in main since 06f026578c4b5fd62a0871c4e400affddf4a0243 and will be in next release
